interfaces/repo.go

```diff
@@ -449,28 +449,73 @@ func (r *Repository) PlugConnections(snapName, plugName string) []*Slot {
 	return result
 }
 
-// Support for sort.Interface
-
-type byPlugSnapAndName []*Plug
+// SlotConnections returns all of the plugs that are connected a given slot.
+func (r *Repository) SlotConnections(snapName, slotName string) []*Plug {
```

Comment:
That makes no sense if Plug has a list of Connections in it. Let's talk online on this one.